scripts/dts/gen_defines.py

```diff
@@ -3,6 +3,7 @@
 # Copyright (c) 2019 - 2020 Nordic Semiconductor ASA
 # Copyright (c) 2019 Linaro Limited
 # SPDX-License-Identifier: BSD-3-Clause
+# SPDX-FileComment: IEC-61508-SIL3
```

Comment:
Although I can see the issue here (this file generates something that basically every application uses), I don' believe I have ever seen any proposals related to the process of safety certification for Python files (or CMake files). I have only read such for C files. What am I missing?

Reply:
I (now) tried to explain in the description concerning the IEC61508 tool classification.
Since gen_defines.py is a code generator (I assume) it is a T3 tool and requires qualification. In a later commit the SPDX-FileComment: reads IEC-61508-T3.